ci/benchmark_parser.py

```diff
@@ -318,15 +326,16 @@ def parse_key_gen_time(result_file):
     return _parse_key_results(result_file, "latency")
 
 
-def get_parameters(bench_id):
+def get_parameters(bench_id, directory="tfhe"):
```

Comment:
need for this default value ?

Reply:
Agreed, I think it's better to pass this argument as a positional one.